freenet/src/main/java/ch/cyberduck/core/freenet/FreenetFindFeature.java

```diff
@@ -15,27 +15,12 @@
  * GNU General Public License for more details.
  */
 
-import ch.cyberduck.core.ListProgressListener;
-import ch.cyberduck.core.Path;
 import ch.cyberduck.core.dav.DAVFindFeature;
 import ch.cyberduck.core.dav.DAVSession;
-import ch.cyberduck.core.exception.BackgroundException;
-import ch.cyberduck.core.shared.DefaultFindFeature;
 
 public class FreenetFindFeature extends DAVFindFeature {
```

Comment:
Isn't this then redundant? If there is nothing left, except the constructor, then DAVFindFeature could be used instead of subclassing.

Reply:
I just keep it around for the integration tests.